docs/src/test/resources/application.conf

```diff
@@ -0,0 +1,3 @@
+akka {
+   loggers = ["akka.testkit.TestEventListener"]
+}
```

Comment:
Probably doesn't matter here, but to be safe always have an empty \n in conf files. 
A missing newline breaks multinode tests due to the way they concat stuff... (so it'd break the perf spec for example etc). Bette safe ;)